Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,40 @@
+public class MockView: Viewable {
+	var ui: Receivable 
+	var guess = ""
+
+	public init(ui: Receivable) {
+		self.ui = ui
+	}
+
+	public func receiveGuess() -> String {
+		guess = ui.getUserInput()
+		if (containsOnlyLetters(guess)) {
+			return guess
+		} else {
+			return "Invalid"
+		}
+	}
+
+	public func assignBlanks(gameWord: String, correctGuesses: [String]) -> String {
+		var blanks: [String] = []
```

Comment:
we didn't discuss a coding style for initializing empty collection variables, but what do you thing about 
```swift
var blanks = [String]()
```